app/uk/gov/hmrc/tradergoodsprofilesdatastore/controllers/GetRecordsController.scala

```diff
@@ -55,22 +55,28 @@ class GetRecordsController @Inject() (
     }
   }
 
-  def getRecord(eori: String, recordId: String): Action[AnyContent] = (identify andThen storeLatest).async {
-    implicit request =>
-      recordsRepository.get(eori, recordId).map {
-        case Some(goodsItemRecords) =>
-          Ok(Json.toJson(goodsItemRecords))
-        case None                   =>
+  def getRecord(eori: String, recordId: String): Action[AnyContent] = identify.async { implicit request =>
+    routerConnector
+      .getRecord(eori, recordId)
+      .map(record =>
+        if (record.active) {
```

Comment:
same as above

Reply:
https://jira.tools.tax.service.gov.uk/browse/TGP-1978 :(

Reply:
and also clarified with ben that getting an inactive record return s a 200 and the inactive record